### PR TITLE
Sponsers

### PR DESCRIPTION
**Pull request by hossam-khalaf**
https://hossam-khalaf.charity.m3ntorship.com/

### CODE REVIEW

File: src/css/sponsors.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.sponsor {
+  @apply px-4 py-2 m-2;
+}
+
+.contact-section__social-icon {
+  @apply bg-c600 opacity-75  w-10 h-10 rounded-full flex justify-center items-center m-1;
```
Comment (MAlngaawy) [inline src/css/sponsors.css:6]: add cursor pointer in social media icons

File: dist/index.html [hunk 2]
```diff
@@ -133,7 +55,107 @@ <h3 class="text-c100 text-md py-3 font-bold">We Nourish</h3>
...
+          </div>
+        </div>
+      </div>
+      <!-- Contact information -->
+      <div class="contact-section bg-c100 py-10">
+        <div class="container">
+          <div class="flex justify-between align-middle py-4 text-c000">
+            <!-- social icons -->
+            <div class="contact-section__social flex flex-1 text-xxs">
+              <div
```
Comment (MAlngaawy) [inline dist/index.html:84]: use link instead of div

File: dist/index.html [hunk 3]
```diff
@@ -133,7 +52,115 @@ <h3 class="text-c100 text-md py-3 font-bold">We Nourish</h3>
     </section>
 
     <section class="sponsors">
-      Sponsors
+      <!-- sponsers logos! -->
+      <div class="container">
+        <div class="flex justify-between mx-auto my-20 flex-wrap">
```
Comment (AbdoMgdy) [inline dist/index.html:57]: my-32 is closer to the psd

File: dist/index.html [hunk 4]
```diff
@@ -133,7 +52,115 @@ <h3 class="text-c100 text-md py-3 font-bold">We Nourish</h3>
...
+                </div>
+                <div class="information">
+                  <p class="leading-normal">needhelp@lovims.com</p>
+                  <small class="information-small leading-normal"
+                    >Email address</small
+                  >
+                </div>
+              </div>
+
+              <div class="contact border-right">
```
Comment (AbdoMgdy) [inline dist/index.html:131]: give custom border color rgba(255,255,255,0.1) from psd

and padding left and right could be 3rem or closer

File: dist/index.html (hunk 2 above)
Comment (hossam-khalaf) [reply dist/index.html:85]: done, thankes for feedbak

File: src/css/sponsors.css (hunk 1 above)
Comment (hossam-khalaf) [reply src/css/sponsors.css]: added automatically when changed to links

File: dist/index.html (hunk 4 above)
Comment (hossam-khalaf) [reply dist/index.html:135]: done

**Pull request merged**